wpeverest/user-registration
# UR-371 Fix - Flatpickr localization issue

### All Submissions:






### Changes proposed in this Pull Request:
Previously Flatpickr localization enqueue through CDN link. This PR solves this problem.

### How to test the changes in this Pull Request:
1. Create a form.
2. Drag and Drop the date field and change the date localization in another language.
3. Check whether localization is working properly or not in the frontend.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Flatpickr localization issue.
